zaas-service/src/test/resources/application-test.yml

```diff
@@ -129,15 +129,15 @@ eureka:
                 routes:
                     api_v1:
                         gatewayUrl: /api/v1
-                        serviceUrl: /gateway
+                        serviceUrl: /zaas
                 apiInfo:
-                    - apiId: zowe.apiml.gateway
+                    - apiId: zowe.apiml.zaas
                       gatewayUrl: api/v1
                       swaggerUrl: https://${apiml.service.hostname}:${apiml.service.port}/api-doc
                       documentationUrl: https://zowe.github.io/docs-site/
                 service:
-                    title: API Gateway
-                    description: API Gateway service to route requests to services registered in the API Mediation Layer and provides an API for mainframe security.
+                    title: ZAAS
+                    description: Zowe Authorization and Authentication Service.
```

Comment:
is it Authorization and Authentication or Authentication and Authorization

Reply:
Zowe Authentication and Authorization Service - fixed